### DISCUSSION TITLE
A little problem with a shortcut

### DISCUSSION THREAD

**Discussion by ovizii** [karakeep-app/karakeep#79] [General]
Most of the time, when hoarder suggests keyboard shortcuts, I just replace the apple key with CTRL being on Windows and it works. Except this time: 

CTRL+E goes straight to Google search inside `Chrome.`

![image](https://github.com/MohamedBassem/hoarder-app/assets/837902/c40a2080-deec-4dc6-aa4b-762b3c764f0b)

**Comment by MohamedBassem** [comment]
hmmm, interesting. Your intuition is correct, when I write `command`, I refer to `ctrl` for windows. Seems like chrome in this case takes precedence. I can't repro because this seems to be a linux/windows specific shortcut according to [this link](https://support.google.com/chrome/answer/157179?hl=en-GB&co=GENIE.Platform%3DDesktop#zippy=%2Caddress-bar-shortcuts%2Ctab-and-window-shortcuts%2Cgoogle-chrome-feature-shortcuts). I think choosing `Ctrl+E` might not have been the wisest idea. I'll google what are some of the "safe" shortcuts to use in web apps and maybe replace that one.
